README.md

```diff
@@ -62,6 +62,41 @@ OpenFisca provides a basic infrastructure, in particular a public instance of it
 hosted on cheap servers.
 As it is free software, anyone can reproduce the OpenFisca infrastructure on its server.
 
+## Project Components
+OpenFisca is a modular project. Depending on your goals, you will install and interact with one or several of the OpenFisca Components.
+
+[![OpenFisca schema](openfisca-doc/schema_openfisca.svg?raw=true)](openfisca-doc/schema_openfisca.svg)
+
+### Web API
+The Web API lets you access the legislation Parameters and Variables.
+
+Example: [Mes Aides](https://mes-aides.gouv.fr) uses the OpenFisca Web API to calculate OpenFisca-France benefits.
+
+- To explore the OpenFisca-France Web API services, use our [French Legistation Explorer](https://legislation.openfisca.fr/)
```

Comment:
Phrasing inconsistency: the rest of the documentation doesn't seem to refer to the provided elements as “ours”, but as “them”: _the_ French legislation explorer would be more appropriate here.